app/driver_auth/views.py

```diff
@@ -69,6 +78,42 @@ def get_oidc_client_list(request):
     return JsonResponse({'clients': CLIENTS.keys()})
 
 
+class DriverSsoAuthToken(APIView):
+    parser_classes = (JSONParser,)
+    permission_classes = (AllowAny,)
+
+    def post(self, request, format=None):
+        token = request.data.get('token')
+        if token:
+            return validate_oauth_token(token)
+        else:
+            return JsonResponse({'error': 'Token parameter is required'}, status=400)
```

Comment:
We're not always consistent about doing this, but I think it would be good to move these status codes to use the constants in the `rest_framework.status` module.